### PR TITLE
Added Soundcloud to blockable services

### PR DESCRIPTION
**Pull request by sandervankasteel**

### CODE REVIEW

File: assets/services.json [hunk 1]
```diff
@@ -1599,6 +1599,15 @@
...
...-27.791 27.778zm102.65-56.589c8.174 0 13.367 3.6399 16.613 9.8659l16.869-7.78c-5.715-11.804-16.103-20.116-33.228-20.116-23.724 0-40.349 19.201-40.349 46.841 0 28.673 15.954 46.83 39.699 46.83 16.486 0 27.385-7.6519 33.611-20.499l-15.305-9.089c-4.801 8.057-9.611 11.559-17.785 11.559-13.623 0-21.541-12.464-21.541-28.801-9e-3 -16.859 7.803-28.811 21.416-28.811zm67.381-16.614h-18.168v90.85h54.6v-17.923h-36.432zm84.294-1.4159c-24.798 0-39.454 19.201-39.454 46.841 0 27.641 14.656 46.83 39.454 46.83 24.777 0 39.443-19.189 39.443-46.83 1e-3 -27.64-14.666-46.841-39.443-46.841zm0 75.652c-14.677 0-20.765-12.591-20.765-28.801 0-16.22 6.088-28.811 20.765-28.811 14.646 0 20.743 12.591 20.743 28.811 0 16.21-6.097 28.801-20.743 28.801zm112.75-23.224c0 14.528-7.12 23.489-18.668 23.489-11.569 0-18.562-9.217-18.562-23.744v-50.757h-18.167v51.013c0 26.459 14.794 41.253 36.729 41.253 23.096 0 36.847-15.188 36.847-41.508v-50.758h-18.179zm69.298-51.012h-28.407v90.85h27.108c23.372 0 46.457-13.506 46.457-45.425 0-33.228-19.2-45.425-45.158-45.425zm-1.299 73.203h-8.951v-55.546h9.611c19.189 0 27.107 9.2061 27.107 27.768 1e-3 16.614-8.695 27.778-27.767 27.778z\" fill=\"url(#a)\" stroke-width=\".99999\"/></svg>"
```
Comment (ameshkov) [inline assets/services.json:1609]: Use this SVG instead please:
```
<svg xmlns="http://www.w3.org/2000/svg"  viewBox="0 0 24 24" fill-rule="evenodd" clip-rule="evenodd"><path d="M19 17.75c2.07 0 3.75-1.68 3.75-3.75 0-2.07-1.68-3.75-3.75-3.75-.173 0-.344.012-.511.035-.73-2.337-2.913-4.035-5.489-4.035-.818 0-1.596.171-2.301.48-.273.119-.449.389-.449.687l0 9.583c0 .414.336.75.75.75l8 0zM7.25 8l0 9c0 .414.336.75.75.75.414 0 .75-.336.75-.75l0-9c0-.414-.336-.75-.75-.75-.414 0-.75.336-.75.75zM4.25 10l0 7c0 .414.336.75.75.75.414 0 .75-.336.75-.75l0-7c0-.414-.336-.75-.75-.75-.414 0-.75.336-.75.75zM1.25 12l0 5c0 .414.336.75.75.75.414 0 .75-.336.75-.75l0-5c0-.414-.336-.75-.75-.75-.414 0-.75.336-.75.75z"/></svg>
```

**Review by ameshkov — CHANGES_REQUESTED** [review]

File: assets/services.json (hunk 1 above)
Comment (sandervankasteel) [reply assets/services.json:1609]: Will do!

Comment (sandervankasteel) [reply assets/services.json]: Updated :)

File: assets/services.json [hunk 2]
```diff
@@ -1599,6 +1599,15 @@
...
     },
+    {
+      "id": "soundcloud",
+      "name": "SoundCloud",
+      "rules": [
+          "||soundcloud.com^",
+          "||sndcdn.com^"
+      ],
+      "icon_svg": "<svg xmlns=\"http://www.w3.org/2000/svg\"  viewBox=\"0 0 24 24\" fill-rule=\"evenodd\" clip-rule=\"evenodd\"><path d=\"M19 17.75c2.07 0 3.75-1.68 3.75-3.75 0-2.07-1.68-3.75-3.75-3.75-.173 0-.344.012-.511.035-.73-2.337-2.913-4.035-5.489-4.035-.818 0-1.596.171-2.301.48-.273.119-.449.389-.449.687l0 9.583c0 .414.336.75.75.75l8 0zM7.25 8l0 9c0 .414.336.75.75.75.414 0 .75-.336.75-.75l0-9c0-.414-.336-.75-.75-.75-.414 0-.75.336-.75.75zM4.25 10l0 7c0 .414.336.75.75.75.414 0 .75-.336.75-.75l0-7c0-.414-.336-.75-.75-.75-.414 0-.75.336-.75.75zM1.25 12l0 5c0 .414.336.75.75.75.414 0 .75-.336.75-.75l0-5c0-.414-.336-.75-.75-.75-.414 0-.75.336-.75.75z\"/></svg>"
```
Comment (Sergey-Lyapin) [inline assets/services.json:1609]: Hello @sandervankasteel
please also add fill=\"currentColor\" parameter

Comment (sandervankasteel) [reply assets/services.json:1609]: @Sergey-Lyapin yes, I will add that :smile:

Comment (sandervankasteel) [reply assets/services.json]: Updated @Sergey-Lyapin.

**Review by Sergey-Lyapin — APPROVED** [review]

**Review by ameshkov — APPROVED** [review]

**Pull request merged**